### DISCUSSION TITLE
Prusa XL profile

### DISCUSSION THREAD

**Discussion by DaTTcz** [supermerill/SuperSlicer#4354] [General]
Hi, does anyone use SupeSlicer for Prusa XL 5 head ? I'm trying to create a profile but it's not working... Can you share your printer profiles ? Thank you.

**Comment by supermerill** [comment]
here one:
[3D Objects.zip](https://github.com/user-attachments/files/16107377/3D.Objects.zip)
I opened ps2.7
I select the printer & print profile i wanted.
I export the config (not bundle)
In SuperSlicer, I `import the prusa config`
Then, i disable the ooze prevention (not yet available in this version) (in print -> multiple extruders)
And I modified the start & end gcode to remove the commands that aren't available in this version (so it starts all extruders, even the ones that aren't used)

I can't test it as I still don't have the printer.

**Reply by DaTTcz** [reply]
Hi, thanks for sharing. I'll try and let you know. I'm looking forward to using SuperSlicer as my main Slicer again :-)

**Reply by DaTTcz** [reply]
Hey, it's just like you wrote. The printers will touch all the heads even if you don't use them. Otherwise, I've tried printing and everything seems good on the first try.

**Comment by DaTTcz** [comment]
Hi, if I upload a gcode file to the cloud (Prusa Connect) and print from it, it prints. However, if I print the same file from the printer menu it crashes at some point during the same time when preparing to print, it restarts and the screen shows a red error message  (PrusaToolChanger-Toolchange to tool that is disabled by tool mapping). I understand that at this stage it's just a modified profile and the difference between the versions is big. But I am grateful anyway.

**Comment by killdaclick** [comment]
@supermerill I would like to do the same for the Prusa MK4 (which are not available in SS), especially the Input Shaper one.
When I export without the bundle and the import it shows a bunch of missing parameters and then all the filament profiles disappear, the plate is missing so practically it does not work? I have got enabled "Show incompatible*" in the options.
How to enable Prusa MK4 profiles?

![image](https://github.com/user-attachments/assets/dbcfe6cd-7c83-42a8-b95c-310b8f15514d)
![image](https://github.com/user-attachments/assets/b4704b76-54d4-4b8c-92ad-d9170db31f20)

**Comment by supermerill** [comment]
I'm now in the process to update the profiels to the 2.7, so with the new variabels the XL& mk4 need.
Will be better.
Also I now have an XL available to test them.

**Reply by DaTTcz** [reply]
Glad to hear a new version is still in the works. If needed I can also test something on Prusa XL 5T.
